src/Vex/GfxBackend.cpp

```diff
@@ -349,8 +362,9 @@ void GfxBackend::OnWindowResized(u32 newWidth, u32 newHeight)
 {
     // Do not resize if any of the dimensions is 0, or if the resize gives us the same window size as we have
     // currently.
-    if (newWidth == 0 || newHeight == 0 ||
-        (newWidth == description.platformWindow.width && newHeight == description.platformWindow.height))
+    if (isSwapchainValid &&
```

Comment:
Resizing with a size of 0 will cause a crash. We should only resize if:
The backbuffer is invalid AND newsize != 0.
And then only if backbuffer is invalid OR newSize != PrevSize.